server/bin/unittests

```diff
@@ -224,7 +224,9 @@ function _dump_logs {
     # Be quiet if there's no database, or if the `dataset` table hasn't been
     # created.
     db=${_testroot}/test_db
-    select='select (select users.username from users where users.id = datasets.owner_id),controller,name,state,(select " " || key || " = (" || length(value) || ") " || substr(value,1,80) from dataset_metadata where dataset_ref = datasets.id) from datasets order by (select users.username from users where users.id = datasets.owner_id),controller,name asc'
+    # select='select (select users.username from users where users.id = datasets.owner_id),controller,name,state,(select " " || key || " = (" || length(value) || ") " || substr(value,1,80) from dataset_metadata where dataset_ref = datasets.id) from datasets order by (select users.username from users where users.id = datasets.owner_id),controller,name asc'
```

Comment:
Remove this

Reply:
I distinctly recall having verified that it was gone but forgotten to resolve the comment. You're right; I remembered incorrectly. So I'm unresolving until my new commit.